engine/src/main/java/org/terasology/engine/TerasologyConstants.java

```diff
@@ -24,18 +24,52 @@
 import java.nio.file.Paths;
 
 /**
+ * This class contains various important file-related constants used by the Terasology engine.
  */
 public final class TerasologyConstants {
+    /**
+     * Name and extension of an entities data file.
```

Comment:
Then the constant could either be removed, or it should be used in the spots you mention. IMO, it would be a good idea to include those fixes in this PR.